flashinfer/gemm/gemm_base.py

```
@@ -5029,6 +5066,141 @@ def forward(
+            self,
+            inputs: List[torch.Tensor],
+            tactic=None,
+            do_preparation: bool = False,
+            **kwargs,
+        ):
+            (a, b, a_descale, b_descale, alpha_tensor, _, out, _, _, _) = inputs
+            m = a.shape[0]
+            k_packed = a.shape[1]
+            n = b.shape[1]
+            real_k = k_packed * 2
+            l = 1
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)